modules/react/table/lib/Table.tsx

```diff
@@ -1,28 +1,241 @@
+          <Table.Header>Table Header</Table.Header>
+          <Table.Header>Table Header</Table.Header>
+        </Table.Row>
+        <Table.Row>
+          <Table.Header>Table Header</Table.Header>
+          <Table.Cell>Table Data Cell</Table.Cell>
+        </Table.Row>
+        <Table.Row>
+          <Table.Header>Table Header</Table.Header>
+          <Table.Cell>Table Data Cell</Table.Cell>
+        </Table.Row>
+      </Table.Body>
+      <Table.Footer>
+        <Table.Row>
+          <Table.Header>Table Header</Table.Header>
+          <Table.Cell>Table Data Cell</Table.Cell>
+        </Table.Row>
+      </Table.Footer>
+    </Table>
+  );
 }
+```
+ */
+export const Table = createComponent('table')({
+  displayName: 'Table',
+  Component: ({children, ...elemProps}: GridProps, ref, Element) => {
+    return (
+      <Element ref={ref} {...mergeStyles(elemProps, tableStencil())}>
+        {children}
+      </Element>
+    );
+  },
+  subComponents: {
+    /**
+     * `Table.Caption` renders a [caption](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/caption) element.
+     *
+     *
+     * ```tsx
+      import {Table} from '@workday/canvas-kit-react/table';
+
+      export default function App() {
+        return (
+          <Table>
+            <Table.Caption>Table Caption</Table.Caption>
+            <Table.Body>
+              <Table.Row>
+                <Table.Header>Table Header</Table.Header>
+                <Table.Cell>Table Cell</Table.Cell>
+              </Table.Row>
+            </Table.Body>
+          </Table>
+        );
+      }
+      ```
+    */
+    Caption: TableCaption,
+    /**
+    * `Table.Head` renders a [thead](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/thead) element.
+    *
+    *
+    * ```tsx
+    import {Table} from '@workday/canvas-kit-react/table';
+
+    export default function App() {
+      return (
+        <Table>
+          <Table.Head>
+            <Table.Row>
+              <Table.Header>Table Header</Table.Header>
+              <Table.Cell>Table Cell</Table.Cell>
+            </Table.Row>
+          </Table.Head>
+        </Table>
+      );
+    }
+    ```
+    */
+    Head: TableHead,
+    /**
+     * `Table.Body` renders a [tbody](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tbody) element.
+     *
+     *
+     * ```tsx
+    import {Table} from '@workday/canvas-kit-react/table';
+
+    export default function App() {
+      return (
+        <Table>
+          <Table.Body>
+            <Table.Row>
+              <Table.Header>Table Header</Table.Header>
+              <Table.Cell>Table Cell</Table.Cell>
+            </Table.Row>
+          </Table.Body>
+        </Table>
+      );
+    }
+    ```
+    */
+    Body: TableBody,
+    /**
+     * `Table.Row` renders a [tr](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tr) element.
+     *
+     * **Note**: `Table.Row` is built on [Grid](/docs/components-layout-grid--basic). It will look for
```

Comment:
is this still true? is it still built on grid?

Reply:
Yup!